lib/vagrant-libvirt/cap/public_address.rb

```diff
@@ -0,0 +1,14 @@
+module VagrantPlugins
+  module ProviderLibvirt
+    module Cap
+      class PublicAddress
+        def self.public_address(machine)
+          # This does not need to be a globally routable address, it
+          # only needs to be accessible from the machine running
+          # Vagrant.
+          machine.ssh_info[:host]
```

Comment:
https://github.com/hashicorp/vagrant/blob/master/plugins/providers/virtualbox/cap/public_address.rb#L6-L10 suggests there should be some rudimentary checks done before attempting to return this. Can you update to follow the same pattern and will then merge

Reply:
Thanks, updated! :muscle: